templates/cisco_nxos_show_interfaces_switchport.textfsm

```diff
@@ -3,7 +3,9 @@ Value SWITCHPORT (.+)
 Value SWITCHPORT_MONITOR (.+)
 Value MODE (.+)
 Value ACCESS_VLAN (\d+)
+Value ACCESS_VLAN_NAME (.*)
```

Comment:
Could you change the regex to `\.+` since I think we're expecting something between the brackets rather than possibly nothing.

Reply:
Agreed will push the updates in a few minutes.